### ISSUE TITLE
About Outputs

### ISSUE THREAD

**Issue by longlongvip** [yu4u/age-estimation-pytorch#11]
I read your code carefully. In train.py, I'm not sure what outputs means. Is it the output of the model FC layer or the Softmax layer? 
Because if it's the output of the FC layer, then it's contradictory to **_ predicted = outputs.max (1)**, and if it's the output of the Softmax layer, it's contradictory to **loss = criterion (outputs, y)**. What's going on? 
Please take the precious time to help me, because I urgently need to get the correct conclusion and thank you for your help.

**Comment by yu4u** [comment]
The output of the model is logits (the output of the FC layer),  not that of softmax.

Please refer to https://pytorch.org/docs/stable/torch.html#torch.max

`outputs.max(1)` is the same as `torch.max(outputs, 1)`.
